src/ontology/components/catbreeds.owl

```diff
@@ -16118,7 +16133,7 @@
     <owl:Axiom>
         <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/VBO_0100262"/>
         <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
-        <owl:annotatedTarget>A group of cats derived from more than one breed of the same species. </owl:annotatedTarget>
+        <owl:annotatedTarget>A cat derived from more than one breed of the same species. </owl:annotatedTarget>
```

Comment:
@matentzn does this look ok?

Reply:
If `species`  refers to the "genus" of cats, then this definition sounds a bit circular (and it has an illegal space in the end by the way)!

ChatGPT suggests something like this:

```
A cat whose ancestry includes a combination of different breeds, without a consistent pedigree linking it to a specific breed standard.
```

But even `A cat whose ancestry includes a combination of different breeds` would make a bit more sense to me? (same for dogs)

Reply:
On the topic of cats, see also this user request: https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/211

for replacing the term "mixed breed cat" with "random bred cat." I don't think that "random bred cat" can replace "mixed breed cat" because they have different definitions, but I think that we can add this new term to VBO.